code/game/objects/items.dm

```diff
@@ -147,22 +147,25 @@
 	return ..()
 
 /obj/item/ex_act(severity, explosion_direction)
+	var/msg = pick(" is destroyed by the blast!", " is obliterated by the blast!", " shatters as the explosion englufs it!", " disentegrates in the blast!", " perishes in the blast!", " is mangled into uselessness from the blast!")
+	if(indestructible)
+		return
```

Comment:
This also prevents indestructible items from being tossed around, which is funnily enough going in the opposite direction of #1271